SPECS/gtk-doc/gtk-doc.spec

```diff
@@ -32,13 +36,20 @@ BuildArch:      noarch
 %description
 The GTK-Doc package contains a code documenter. This is useful for extracting
 specially formatted comments from the code to create API documentation.
+
 %prep
 %setup -q
+
+# Move this doc file to avoid name collisions	
+mv doc/README doc/README.docs
+
 %build
-%configure --disable-silent-rules CFLAGS="%{build_cflags}"
-make %{?_smp_mflags}
+%meson -Dtests=false
```

Comment:
Could you also check if we don't need to update the `%check` section as well now?